content/4. Deeltaken/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer & Uint.md

```diff
@@ -0,0 +1,226 @@
+---
+title: 1. Integer
+taxonomie:
+  - ib-19.2.Datatype-Primitief-Integer.DT
+  - ib-19.3.Datatype-Primitief-Integer.DT
+  - ib-19.2.Datatype-Primitief-Uint.DT
+  - ib-19.3.Datatype-Primitief-Uint.DT
+---
+
+> Meer informatie over [[content/2. Ondersteunende-informatie/3. Processen/4. Implementatieproces/2. Bouwen softwaresysteem/Integer/1. Integer|Integer]]
+
+> Meer informatie over [[2. Uint|Uint]]
+
+## Opdracht 1
+Voeg het initialiseren toe aan het volgende codeblok. Maak gebruik van het integer datatype.
+
+### Specificaties:
+- Maak een variabele `age`.
+- Geef `age` de waarde `23`.
+- Print `age`.
+
+### Verwachte output:
+Als `age = 23, komt in de console:
+```
+De leeftijd is: 23
+```
+
+### Nu jij:
+```csharp runner
+using System;
+
+age = 23;
+
+Console.WriteLine("De leeftijd is: " + age);
+```
+
+> [!info]- Mogelijke uitwerking
+> ``` csharp
+> using System;
+> 
+> int age;
+> 
+> age = 23;
+> 
+> Console.WriteLine("De leeftijd is: " + age);
+> ```
+
+---
+
+## Opdracht 2
+Gebruik bij de volgende casus een integer.
```

Comment:
Opmerking: Dit is eerder een hint of regel/eis/ding i.p.v. een opdrachtomschrijving. Is de opdracht hetzelfde als de vorige? Dan zou ik lekker gewoon die tekst hier repurposen.